grammars/level5-Additions.lark

```diff
@@ -1,4 +1,6 @@
-program: _EOL* (command | error_invalid)* (_SPACE)* (_EOL+ (command | error_invalid) (_SPACE)*)* _EOL* //lines may end on spaces and might be separated by many newlines
+program: _empty_program | _non_empty_program
+_empty_program: (_EOL | _SPACE)*
```

Comment:
Smart fix @boryanagoncharenko!!

Reply:
It is the only fix that actually solved the problem. I thought I understood why there is an ambiguity at level 6, but as I dived deeper I figured the ambiguity should already kick in at level 5. But it did not, which made me wonder whether I understood what's going wrong in the first place... So, I had to take this alternative approach. Grammar ambiguities are hard.